pkg/domain/dogu_test.go

```diff
@@ -38,7 +38,7 @@ func Test_TargetDogu_validate_missingVersionOkayForAbsentDogu(t *testing.T) {
 }
 
 func Test_TargetDogu_validate_defaultToPresentState(t *testing.T) {
-	dogu := Dogu{Name: officialDogu1, Version: version1_2_3}
+	dogu := Dogu{Name: officialDogu1, Version: version123}
```

Comment:
I would not be so picky in this case. We can just use major versions 1,2,3,4,5.. because we do not test the functionality if a version is higher, equal or lower here.

Reply:
Yes, this is no real problem.